package proto_ff;

import "yd_fieldoptions.proto";


message E_FootprintFootprintAttributeDesc
{
	optional int32 m_value = 1[(yd_fieldoptions.field_cname) = "值"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "类型"];
}

message E_FootprintFootprintActiveattributeDesc
{
	optional int32 m_value = 1[(yd_fieldoptions.field_cname) = "值"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "类型"];
}

message E_FootprintFootprint
{
	optional int64 m_ID = 1[(yd_fieldoptions.field_cname) = "足迹ID"];
	optional string m_name = 2[(yd_fieldoptions.field_cname) = "足迹名称", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_type = 3[(yd_fieldoptions.field_cname) = "外观类型"];
	optional int32 m_quality = 4[(yd_fieldoptions.field_cname) = "足迹品质"];
	optional string m_professionLimit = 5[(yd_fieldoptions.field_cname) = "职业限制", (yd_fieldoptions.field_bufsize) = 64];
	optional int64 m_activationItem = 6[(yd_fieldoptions.field_cname) = "激活道具ID"];
	optional int32 m_activationNum = 7[(yd_fieldoptions.field_cname) = "激活道具数量"];
	optional int64 m_starId = 8[(yd_fieldoptions.field_cname) = "升星道具id"];
	optional string m_starNum = 9[(yd_fieldoptions.field_cname) = "升星道具数量", (yd_fieldoptions.field_bufsize) = 256];
	optional int32 m_starUp = 10[(yd_fieldoptions.field_cname) = "升星等级上限"];
	optional int32 m_starBer = 11[(yd_fieldoptions.field_cname) = "每次升星属性增加的百分比值"];
	repeated E_FootprintFootprintAttributeDesc m_attribute = 12[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 6];
	repeated E_FootprintFootprintActiveattributeDesc m_ActiveAttribute = 13[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 6];
}


message Sheet_FootprintFootprint
{
	repeated E_FootprintFootprint E_FootprintFootprint_List = 1[(yd_fieldoptions.field_arysize)=8];
}
